src/actions/post-install.ts

```diff
@@ -0,0 +1,13 @@
+import picocolors from 'picocolors';
+
+export async function showNextSteps(): Promise<void> {
+	console.log(`
+${picocolors.green(
+	'You can now lint your CSS files by running the command:\nnpx stylelint "**/*.css"',
```

Comment:
> the command:\nnpx stylelint

This part will probably have to be package specific.

e.g. for `yarn` you will have to read yarnpkg/yarn#3937 and https://yarnpkg.com/blog/release/2.0#new-command-yarn-dlx

Reply:
I originally use pnpm for all projects, and have occasionally encountered situations where `pnpm <command>` worked without `dlx`, even though the documentation required `pnpm dlx <command>`. I haven't gone deep into the details of this behavior, but I'll add it to the list to look into. It's probably specific to each package manager. That said, I believe explicitness is better than implicitness - using `pnpm dlx` is a more clear variant for me